### PR TITLE
Fix color package

### PR DESCRIPTION
**Pull request by mtelvers**
Followup to https://github.com/ocaml/opam-repository/pull/26927

### CODE REVIEW

File: packages/color/color.0.3.0/opam [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+depends: [
+  "dune" {>= "2.0"}
+  "ocaml" {>= "4.08.0"}
+  "gg" {>= "1.0.0"}
+  "alcotest" {with-test}
+  "ocaml-lsp-server" {with-dev-setup}
+  "ocamlformat" {= "0.26.2" & with-dev-setup}
+]
+build: [
+  ["dune" "subst"] {pinned}
```
Comment (shonfeder) [inline packages/color/color.0.3.0/opam:22]:
```suggestion
  ["dune" "subst"] {dev}
```

**Comment by ElectreAAS** [comment]
For the record here: the problem with #26927 was that line `url: {` instead of `url {`, is that correct?

**Comment by shonfeder** [comment]
That's a funky quirk of the opam format.

**Comment by mseri** [comment]
Thanks!

**Pull request merged**